distro/exporter.go

```diff
@@ -61,21 +62,33 @@ func newOTLPExporter(c *exporterConfig) (trace.SpanExporter, error) {
 	return otlptracegrpc.New(context.Background(), opts...)
 }
 
+func otlpEndpoint(configured string) string {
+	if configured != "" {
+		return configured
+	}
+
+	// Allow the exporter to interpret these environment variables directly.
+	envs := []string{otelExporterOTLPEndpointKey, otelExporterOTLPTracesEndpointKey}
+	for _, env := range envs {
+		if _, ok := os.LookupEnv(env); ok {
+			return ""
+		}
+	}
+
+	// Use the realm only if OTEL_EXPORTER_OTLP*_ENDPOINT are not defined.
+	if realm, ok := os.LookupEnv(splunkRealmKey); ok {
```

Comment:
Empty (`""`) and `"none"` env var values should be interpreted as if the realm is not defined.

The same comment applies to `func jaegerEndpoint`

References:
- https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#parsing-empty-value
- https://github.com/signalfx/gdi-specification/blob/main/specification/configuration.md#instrumentation-libraries

Reply:
Updated. PTAL